src/responsibleai/rai_analyse/create_score_card.py

```diff
@@ -147,7 +147,7 @@ def validate_and_correct_config(config, insight_data):
             f for f in fc["sensitive_features"] if f not in i_data.categorical_features
         ]
         _logger.warning(
-            "Non categorical features in fairness dropped: {}".format(dropped_features)
+            "Non categorical features dropped for fairness assessment: {}".format(dropped_features)
```

Comment:
This is unrelated but clearly an improvement on the logged text, so I added it in this PR. If you strongly feel like this warrants its own PR please comment.